internal/blockchain/ethereum/ethereum.go

```diff
@@ -199,13 +202,24 @@ func (e *Ethereum) Init(ctx context.Context, cancelCtx context.CancelFunc, conf
 	}
 	e.cache = cache
 
-	e.streams = newStreamManager(e.client, e.cache)
-	batchSize := ethconnectConf.GetUint(EthconnectConfigBatchSize)
-	batchTimeout := uint(ethconnectConf.GetDuration(EthconnectConfigBatchTimeout).Milliseconds())
-	stream, err := e.streams.ensureEventStream(e.ctx, e.topic, batchSize, batchTimeout)
+	e.streams = newStreamManager(e.client, e.cache, e.ethconnectConf.GetUint(EthconnectConfigBatchSize), uint(e.ethconnectConf.GetDuration(EthconnectConfigBatchTimeout).Milliseconds()))
+
+	e.backgroundStart = e.ethconnectConf.GetBool(EthconnectBackgroundStart)
+	if e.backgroundStart {
+		e.backgroundRetry = &retry.Retry{
+			InitialDelay: fftypes.ParseToDuration(defaultBackgroundInitialDelay),
```

Comment:
I think these should be config settings @EnriqueL8

Reply:
Okay, I did think about it but wasn't 100% sure :) - will do it!